lib/wallet/open_channel.dart

```diff
@@ -87,7 +87,7 @@ class _OpenChannelState extends State<OpenChannel> {
                       crossAxisAlignment: CrossAxisAlignment.start,
                       children: [
                         Text(
-                            "Define the amount of bitcoin you want to lock into the Lightning channel. 10101 will double it for great inbound liquidity. Max channel size is currently $takerChannelAmount sats.",
+                            "Define the amount of bitcoin you want to lock into the Lightning channel. 10101 will double it for great inbound liquidity. The max amount you can use is currently $takerChannelAmount sats.",
```

Comment:
❓ Isn't this text dynamic with the amount the user inputs? It still reads weird to me, because it seems to imply that the user cannot go over this number, but they literally can.

Reply:
Good point, I'll add the balance as validation constraint as well.

Reply:
https://github.com/itchysats/10101/pull/517/commits/0da990490f9b5c4f0aa24478082441ba5db3fcd4

Note: I think the balance does not *have* to be confirmed to open a channel, but getting the logic for pending *or* confirmed right might introduce regression bugs so I opted for `confirmed` right now.

Reply:
We could actually introduce form validation like for the CFD amount here, but I am not sure it should get priority

Reply:
Hmm, but I think we already achieve that with [line 37](https://github.com/itchysats/10101/pull/517/commits/7fba080297b362580597c80f1ee88144303dba24#diff-f66cc5c065055955fc377c471cc79795967d0fa0e2fdc56a0153e71ebeb747a1L37).

What I was referring to, is that the text changes with the value that the user inputs e.g.

```
The max amount you can use is currently 50 sats.
```

I think what we are actually trying to say here is that the _maximum_ outbound capacity that we allow is 500_000 sats (based on `OpenChannel.maxChannelAmount`). Or am I mistaken?

Otherwise it reads weird to me. The number changes as the user inputs the very same number just below.

https://user-images.githubusercontent.com/9418575/205525727-e26aff39-faea-459e-918d-9ef636adcb23.mp4

Reply:
Fair enough, I'll change the value that we display to be the balance, capped by the maximum amount